playbooks/roles/nginx-proxy/templates/nginx-proxy.j2

```diff
@@ -13,6 +13,24 @@ server {
 
     {{ NGINX_PROXY_EXTRA_CONFIG }}
 
+    set $expected_origin false;
+    if ($http_origin ~ {{ NGINX_PROXY_ORIGIN_REGEX }}) {
+    set $expected_origin true;
+    }
+
+    {% for response_code in NGINX_PROXY_CORS_HEADERS_FOR_RESPONSE_CODES %}
```

Comment:
```suggestion
    {% for response_code in NGINX_PROXY_CORS_HEADERS_FOR_RESPONSE_CODES -%}
```

This adds whitespace control tag to the closing bracket so that the indentation doesn't get messed up.